tests/lib/rules/require-unicode-regexp.js

```diff
@@ -36,6 +36,7 @@ ruleTester.run("require-unicode-regexp", rule, {
         "new RegExp('', flags)",
         "function f(flags) { return new RegExp('', flags) }",
         "function f(RegExp) { return new RegExp('foo') }",
+        "function f(patternAndFlags) { return new RegExp(...patternAndFlags) }",
```

Comment:
good to have a test like `RegExp(...patternAndFlags)` (as you described in the PR desc).

otherwise LGTM! 👍

Reply:
Thanks, I've added that test.